src/test/scripts/helm_install.sh

```diff
@@ -47,6 +47,12 @@ helm install -n "${TARGET_NAMESPACE}" --wait \
    --version "$POSTGRES_CHART_VERSION" \
    bitnami/postgresql > $LOG_DOWNLOAD_DIR/helm_install_log.txt
 
+if [ -n "$2" ]
+then
+  kubectl cp $2 $TARGET_NAMESPACE/$POSTGRES_RELEASE_NAME-0:/tmp/db-init-script.sql
+  kubectl exec --stdin --tty $POSTGRES_RELEASE_NAME-0 -- /bin/bash -c "psql postgresql://$PRODUCT_NAME:$PRODUCT_NAME@localhost:5432/$PRODUCT_NAME -f /tmp/db-init-script.sql"
```

Comment:
I think running `psql` with params rather than a url makes it easier to understand which `$PRODUCT_NAME` is which:

```
kubectl exec ${POSTGRES_RELEASE_NAME}-0 -- /bin/bash -c "PGPASSWORD=$PRODUCT_NAME psql --host 127.0.0.1 -U $PRODUCT_NAME -d $PRODUCT_NAME -p 5432 -f /tmp/db-init-script.sql"
```